felix/bpf-gpl/include/libbpf/src/btf.c

```diff
@@ -2828,7 +2828,7 @@ static int btf_dedup_remap_types(struct btf_dedup *d);
  * types, in general, can form graphs containing cycles, not just DAGs.
  *
  * While algorithm does deduplication, it also merges and resolves type
- * information (unless disabled throught `struct btf_opts`), whenever possible.
+ * information (unless disabled thought `struct btf_opts`), whenever possible.
```

Comment:
Should be "through"

Reply:
Also BTW the upstream for felix/bpf-gpl/include/libbpf is in the kernel: see the README at https://github.com/libbpf/libbpf.  Hence these fixes are likely to be lost if we sync with a newer version of upstream, and you may like to look at contributing the fixes upstream as well.

Reply:
Just as I thought. I will close this PR and will open against the libbpf. Thanks for the advice.